src/store/DataContext.js

```diff
@@ -0,0 +1,75 @@
+import { createContext, useState, useEffect } from "react";
+
+//creates the context and initial values are set
+const DataContext = createContext({
+    input: "",
+    tasks: [],
+    handleInput: (e) => {},
+    handleSetTasks: () => {},
+    deleteCompletedTask: (clickedObj) => {},
+    toastStatus: false,
+});
+
+// creates context provider
+export function DataContextProvider(props) {
```

Comment:
Use `dispatch` for updating values from context

Reply:
https://kentcdodds.com/blog/how-to-use-react-context-effectively

Reply:
https://www.youtube.com/watch?v=uXWycyeTeCs

Reply:
done